lib/auth/auth.go

```diff
@@ -3727,6 +3727,10 @@ func (a *Server) NewWebSession(ctx context.Context, req types.NewWebSessionReque
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+	if req.LoginIP == "" {
+		// TODO(antonam): consider turning this into error after all use cases are covered (before v14.0 testplan)
+		log.Warnf("Creating new web session without login IP specified.")
```

Comment:
I think if IP pinning is enabled they will get an error from `generateUserCert` right? So this warning will just be noise for people without ip pinning enabled. I don't think we should log until we can actually return an error instead.

I suggested making it an error to make sure that all cases where we need to set the ip are covered, whether or not ip pinning is actually enabled so that we will actually catch it in tests, but since there are still some known uncovered cases I don't think it helps

Reply:
Changed log level to `debug`.